pages/tutorials/github_merge_queue.md

```diff
@@ -16,6 +16,12 @@ The merge queue creates temporary branches with a special prefix to validate pul
 
```

Comment:
I can't comment on it, but let's update the ordering of the intro paragraph for the steps on line 15 to align with the updated instructions. Thinking:

```
The merge queue creates temporary branches with a special prefix to validate pull request changes. To avoid these temporary branches triggering multiple builds on the same pipeline for the same commit, we recommend enabling the option to skip builds on existing commits.

If you already use branch limiting rules in the pipeline, you must also add a filter to match branches with the special prefix.
```